sympy/vector/basisdependent.py

```diff
@@ -269,6 +269,11 @@ def _new(cls, *args, **options):
                 expr = arg
             elif isinstance(arg, (Cross, Dot, Curl, Gradient)):
                 extra_args.append(arg)
+            elif isinstance(arg, Vector):
+                try:
+                    arg.components
+                except AttributeError:
+                    raise ValueError("Wrong way of defining a vector.")
```

Comment:
This isn't a very descriptive error message?
```suggestion
               except AttributeError as exc:
                    raise ValueError("Wrong way of defining a vector.") from exc
```
At least attach the exception cause

Reply:
Error message has been improved and the exception cause has been attached now. 

Thank you for taking the time to review my code!